pkg/securityscan/alert/prometheusrule.go

```diff
@@ -0,0 +1,74 @@
+package alert
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/sirupsen/logrus"
+	meta1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	k8Yaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	cisoperatorapiv1 "github.com/rancher/cis-operator/pkg/apis/cis.cattle.io/v1"
+	"github.com/rancher/wrangler/pkg/name"
+)
+
+func NewPrometheusRule(clusterscan *cisoperatorapiv1.ClusterScan, clusterscanprofile *cisoperatorapiv1.ClusterScanProfile, imageConfig *cisoperatorapiv1.ScanImageConfig) (*monitoringv1.PrometheusRule, error) {
+	configdata := map[string]interface{}{
+		"namespace":       cisoperatorapiv1.ClusterScanNS,
+		"name":            name.SafeConcatName("rancher-monitoring-cis-alerts", clusterscan.Name),
```

Comment:
I don't think we should use `rancher-monitoring` since its a resource created by `rancher-cis`. Using `rancher-monitoring` would make it seem like its a resource owned by the other chart.

```suggestion
		"name":            name.SafeConcatName("rancher-cis-alerts", clusterscan.Name),
```